receiver/httpdreceiver/metadata.yaml

```diff
@@ -0,0 +1,70 @@
+name: httpdreceiver
+
+labels:
+  server_name:
+    description: The name of the Apache HTTP server
+  workers_state:
+    value: state
+    description: The state workers
+    enum:
+      - busy
+      - idle
+  scoreboard_state:
+    value: state
+    description: The state of a connection
+    enum:
+      - open
+      - waiting
+      - starting
+      - reading
+      - sending
+      - keepalive
+      - dnslookup
+      - closing
+      - logging
+      - finishing
+      - idle_cleanup
+
+metrics:
+  httpd.uptime:
+    description: The amount of time that the server has been running in seconds
+    unit: s
+    data:
+      type: sum
+      monotonic: true
+      aggregation: cumulative
+    labels: [ server_name ]
+  httpd.current_connections:
+    description: The number of active connections currently attached to the HTTP server
+    unit: connections
+    data:
+      type: gauge
+    labels: [ server_name ]
+  httpd.workers:
+    description: The number of workers currently attached to the HTTP server
+    unit: connections
+    data:
+      type: gauge
```

Comment:
sum/non-monotonic/cumulative?

Reply:
The idea is that by having this as a non-monotonic sum, in case of a future aggregation like what you described across multiple servers, or any other metric attribute (if any) the consumer knows how to do this aggregation without any extra input from the user. In case of a "gauge" the aggregation is unknown so user will also have to provide an aggregation function to be used when aggregating across any dimension.